server/raft.go

```diff
@@ -3680,11 +3674,11 @@ func (n *raft) writePeerState(ps *peerState) {
 	if bytes.Equal(n.wps, pse) {
 		return
 	}
-	// Stamp latest and kick writer.
+	// Stamp latest and write the peer state file.
 	n.wps = pse
```

Comment:
If we write inline do we need to store []byte anymore?

Reply:
Only reason I can think of is so we don't rewrite it to disk if it hasn't changed, otherwise we could remove it.

Reply:
I don't think we compare it anymore do we?

Reply:
I think we do with the `bytes.Equal` checks above.

Reply:
Yes you are correct. Apologies.